data/src/main/java/org/apache/iceberg/data/DeleteFilter.java

```diff
@@ -58,11 +58,11 @@
 
 public abstract class DeleteFilter<T> {
   private static final Logger LOG = LoggerFactory.getLogger(DeleteFilter.class);
-  private static final long DEFAULT_SET_FILTER_THRESHOLD = 100_000L;
+  private static final long DEFAULT_STREAM_FILTER_THRESHOLD = 100_000L;
   private static final Schema POS_DELETE_SCHEMA =
       new Schema(MetadataColumns.DELETE_FILE_PATH, MetadataColumns.DELETE_FILE_POS);
 
-  private final long setFilterThreshold;
+  private final long streamFilterThreshold;
```

Comment:
I think calling this the streamFilterThreshold is more appropriate, since it is the threshold past which we use the streaming delete filter.